lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -364,12 +387,20 @@ spec = parallel $ do
             let dlg = WalletDelegation
                     {active = NotDelegating, next = [next1]}
             W.guardQuit dlg (Coin 0) `shouldBe` Right ()
-     where
-         pidA = PoolId "A"
-         pidB = PoolId "B"
-         pidUnknown = PoolId "unknown"
-         knownPools = Set.fromList [pidA, pidB]
-         next = WalletDelegationNext
+
+    parallel $ describe "Migration" $ do
+        describe "migrationPlanToSelectionWithdrawals" $ do
+            it "Target addresses are cycled correctly." $
+                property prop_migrationPlanToSelectionWithdrawals_addresses
+            it "Inputs and outputs are preserved in the correct order." $
+                property prop_migrationPlanToSelectionWithdrawals_io
+
+  where
+    pidA = PoolId "A"
+    pidB = PoolId "B"
+    pidUnknown = PoolId "unknown"
+    knownPools = Set.fromList [pidA, pidB]
+    next = WalletDelegationNext
```

Comment:
This `where` clause is a bit annoying: it should belong to the pool delegation tests, but its scope covers the entire property test suite for this module.